apps/core/priv/repo/migrations/20180213103123_add_money_with_currency_type_to_postgres.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Core.Repo.Migrations.AddMoneyWithCurrencyTypeToPostgres do
+  use Ecto.Migration
+
+  def up do
+    execute """
+      CREATE TYPE public.money_with_currency AS (currency_code char(3), amount numeric(20,8))
```

Comment:
can you give a doc link or something for this so that everyone can have a look at it. @oyeb